client/src/pages/profile.tsx

```diff
@@ -49,31 +52,23 @@ const useStyles = makeStyles(() => ({
 
 function ProfilePage(props: { accessToken: string }): JSX.Element {
   const classes = useStyles();
-  const {
-    editedData: editedMentor,
-    error: mentorError,
-    isLoading: isMentorLoading,
-    isSaving: isMentorSaving,
-    isEdited: isMentorEdited,
-    editData: editMentor,
-    saveMentorDetails,
-  } = useWithMentor(props.accessToken);
+  const { state, saveMentor, editMentor } = useWithMentor(props.accessToken);
```

Comment:
There really shouldn't be an accessToken available to javascript at all at this point...but if the app is broken in the sense that we still use it even having cookies, the let's isolate it as much as possible.

It's 100% wrong to have components need to know there's such a thing as an accessToken and then need to pass it back to hooks.

Make sure the accessToken is available in redux for the time being and then let `useWithMentor` just get it from `useSelector` until we fully stamp it out